registry/Scripts/pre-reqs.ps1

```diff
@@ -160,31 +164,33 @@ function New-StorageAccount (
                     -StorageAccountBlobContainer "images"
 #>
 function New-StorageAccountContainer (
-    [Parameter(Mandatory = $true, HelpMessage = "Name of the resource group under which stroage account to be created.")]
+    [Parameter(Mandatory = $true, HelpMessage = "Name of the resource group under which storage blob to be created.")]
```

Comment:
I would say that `The storage account's resource group` is good enough

Reply:
updated.